charts/ecosystem/templates/api.yaml

```diff
@@ -117,8 +117,16 @@ spec:
           value: {{ include "ecosystem.host.url" . }}/api
         - name: GALASA_USERNAME_CLAIMS
           value: {{ join "," .Values.dex.usernameClaims | quote }}
+        - name: GALASA_GLOBAL_REQUEST_CAPACITY
```

Comment:
Are these needed if that code PR never got delivered ?

Reply:
Good spot, thanks! Removed in https://github.com/galasa-dev/helm/pull/63/commits/427e95b613da8dd5f04d8bb8fbbe58f31623038a